CMakeLists.txt

```
@@ -432,6 +434,9 @@ endif()
 if(NOT(PYTHONINTERP_FOUND))
   message(SEND_ERROR "You must install Python to compile DDNet")
 endif()
+if(NOT(SQLite3_FOUND))
+  message(SEND_ERROR "You must install SQLite3 to compile the DDNet server")
```

```suggestion
  message(SEND_ERROR "You must install SQLite3 to compile DDNet")
```

We don't have an option to only compile the server.

---

*We don't have an option to only compile the client.